micro-rdk/src/esp32/i2c.rs

```diff
@@ -4,9 +4,12 @@ use crate::common::config::{AttributeError, Kind};
 use crate::common::i2c::{I2CErrors, I2CHandle};
 use crate::esp32::esp_idf_svc::hal::delay::BLOCK;
 use crate::esp32::esp_idf_svc::hal::gpio::AnyIOPin;
-use crate::esp32::esp_idf_svc::hal::i2c::{I2cConfig, I2cDriver, I2C0, I2C1};
+use crate::esp32::esp_idf_svc::hal::i2c::{I2cConfig, I2cDriver, I2C0};
 use crate::esp32::esp_idf_svc::hal::units::Hertz;
 
+#[cfg(not(any(esp32c3, esp32c2, esp32c6)))]
```

Comment:
Until we have board support in our architecture, this is how we will need to do it.